maas-pro.md

```diff
@@ -966,6 +990,12 @@ curl https://partners.voiapp.io/v1/vehicles/id/12345678-1337-abcd-1234-1234abcd0
         "code": "D0IT",
         "zoneId": "145",
         "locked": true
+      },
+      "price": {
+        "currency": "EUR",
+        "pricePerMinute": 15,
```

Comment:
15 EUR per minute and starting price of 100 EUR look huge :) Might be better to set some realistic values to avoid potential misunderstandings

Reply:
it is cents ;)

Reply:
also copy-pasted from example above ;)